pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -241,6 +242,150 @@ func (plugin *awsElasticBlockStorePlugin) ConstructVolumeSpec(volName, mountPath
 	return volume.NewSpecFromVolume(awsVolume), nil
 }
 
```

Comment:
It will be nice to have all the block volume support for EBS functions/interfaces - moved to a separate file called `ebs_block_device.go` or something like that